server/src/conf/logback.xml

```diff
@@ -29,9 +29,17 @@
         </encoder>
     </appender>
 
+    <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
nit: You might want to keep the naming schema of the appenders consistent within this file. The file appended is called `FILE`, so a console appended could have a name like `CONSOLE` or `STDOUT`.